scripts/multisig/multisig_info.json

```diff
@@ -1,13 +1,15 @@
 {
   "agentInstances": [
-    "0x31D3202d8744B16A120117A053459DDFAE93c855",
-    "0x62815Bcc2941394801B2DA338568afdf93959E7c"
+    "0xD64bf8A8C9a9cbf43aA80E9A8FfeeaFE8a758088",
```

Comment:
This is a setup that proved the service multisig changes work wen utilizing the script